### PR TITLE
📖 Add instructions to reproduce validator tests

### PR DESCRIPTION
**Pull request by antiphoton**

### CODE REVIEW

**Review by Gregable — APPROVED** [review]

**Comment by Gregable** [comment]
Thanks!

File: validator/README.md [hunk 1]
```diff
@@ -123,3 +123,31 @@ $ python
...
+1. Start an interacitve docker container
+    ```bash
+    docker run -it ubuntu bash
+    ```
+1. Run following commands in the container. Note that you are already the `root` user inside the docker container.
+    ```bash
+    apt update
+    apt install -y curl git gpg sudo
+    curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
+    apt install -y nodejs
```
Comment (twifkak) [inline validator/README.md:141]: Is it possible to simplify this via https://hub.docker.com/_/node?

Something like (untested):
```bash
docker run -it node:16 bash
```

and

```
apt install -y git
```

File: validator/README.md [hunk 2]
```diff
@@ -123,3 +123,31 @@ $ python
 ```
 
 Now `cd amphtml/validator` and run `python build.py`.
+
+### Locally Reproduce Validator Tests of Circle CI workflow
+
+In the case the test passes in your local machine but fails in Circle CI,
+you can use docker to reproduce the test errors.
+
+1. Start an interacitve docker container
```
Comment (honeybadgerdontcare) [inline validator/README.md:132]: s/interacitve/interactive

**Review by honeybadgerdontcare — APPROVED** [review]

File: validator/README.md (hunk 1 above)
Comment (antiphoton) [reply validator/README.md]: Done

File: validator/README.md (hunk 2 above)
Comment (antiphoton) [reply validator/README.md]: Done

**Pull request merged**